administration/package.json

```diff
@@ -54,8 +54,8 @@
     "typescript": "^4.8.3"
   },
   "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
+    "start": "set -ae; REACT_APP_APPLICATION_COMMIT=$(./application_commit.sh); react-scripts start",
```

Comment:
Move `set -ae` to the script.

Reply:
The `set -ae` is necessary here, to make the script exit if it fails. Putting it into the script has no effect.

Reply:
We could also do something like:

```suggestion
    "start": "c=$(./application_commit.sh) && REACT_APP_APPLICATION_COMMIT=$c react-scripts start",
```

That seems to work for me as well. Is that better?

Reply:
@maxammann lmk what you think. You can also merge this otherwise.

Reply:
Hm I wonder why that works but the initial version without the `set`

Just tested on macOS with zsh, the set version works so lets merge it.